tox.ini

```diff
@@ -1,7 +1,19 @@
 [tox]
-envlist=py35,py36
-
+envlist=py35,py36,py37
 [testenv]
 deps=pytest
+     pytest-cov
+     flake8
+     flake8-import-order
+whitelist_externals=
```

Comment:
Just curious: what does this directive do?

Reply:
Tox is very strict. Only commands in the virtualenv are allowed. So running system bash, sed or grep will give a failure. Unless you whitelist them.